src/main/java/com/nike/vault/client/VaultClient.java

```diff
@@ -287,6 +305,7 @@ protected Response execute(final HttpUrl url, final String method, final Object
         try {
             return gson.fromJson(response.body().string(), responseClass);
         } catch (IOException|JsonSyntaxException e) {
+            logger.debug(LOG_HTTP_RESPONSE_BODY_TMPL, gson.toJson(response.body()));
```

Comment:
@tlisonbee, @sdford Maybe we should make this INFO level, it would be useful because if a client gets blocked by CloudFront you would be able to see in the response body here?